src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OpprettOppgaverForBarnService.kt

```diff
@@ -55,8 +55,9 @@ class OpprettOppgaverForBarnService(private val oppgaveClient: OppgaveClient,
                                                     aktørId = aktørId,
                                                     oppgavetype = Oppgavetype.InnhentDokumentasjon)
         val oppgaveBeskrivelser = oppgaveClient.hentOppgaver(finnOppgaveRequest)
-                ?.mapNotNull { it.beskrivelse }
-                ?.map { it.trim().lowercase() }
+                .mapNotNull { it.beskrivelse }
+                // Burde denne filtrere vekk allerede lukkede oppgaver? Sånn att vi ikke risikerer att vi finner en gammel oppgave? Og då har match mot den
```

Comment:
Men da finnes den uansett allerede, og da skipper vi opprettelse ?

Reply:
Hvis det er en oppgave for 6 mnd så vil beskrivelsen være forskjellig, sammenlignet med beskrivelse for 1 år